src/core/client.js

```diff
@@ -158,7 +158,11 @@ export class Client {
    * Set the active user
    */
   setActiveUser(activeUser) {
-    return this.activeUserStorage.set(`${this.appKey}.${ACTIVE_USER_KEY}`, activeUser);
+    if (activeUser != null) {
```

Comment:
question that I had was if should use `!=` or `!==` in this case

Reply:
you can just do `if (activeUser) {`

Reply:
done! thanks @thomasconner